### PR TITLE
Counter supports min and max props

### PR DESCRIPTION
**Pull request by lfender6445**
Add min and max props to `<Counter />`

### CODE REVIEW

File: packages/react-ui-core/src/Counter/Counter.js [hunk 1]
```diff
@@ -67,8 +92,9 @@ export default class Counter extends Component {
           >{`${decrementUnit} ${count} ${incrementUnit}`}</span>
           <span
             role="presentation"
-            onClick={() => this.handleClick(this.state.count - changeValue)}
+            onClick={this.increment}
             className={cn(
+              'foo',
```
Comment (tadjohnston) [inline packages/react-ui-core/src/Counter/Counter.js]: Whoops!

**Comment by tadjohnston** [comment]
I think you missed changing the `handleClick` for the stories.

**Comment by tadjohnston** [comment]
Love the changes @lfender6445 !!!

**Comment by tcwilson** [comment]
👍

**Comment by lfender6445** [comment]
@tadjohnston good catches updated

**Pull request merged**